examples/kubernetes/cilium-config.yaml

```diff
@@ -0,0 +1,43 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cilium-config
+  namespace: kube-system
+data:
+  # If you do not want to run cilium in debug mode change it to false
+  debug: "true"
+
+  # This etcd-config contains the etcd endpoints of your cluster. If you use
+  # TLS please make sure you uncomment the ca-file
+  etcd-config: |-
+    ---
+    endpoints:
+    - http://127.0.0.1:2379
+    #
+    # In case you want to use TLS in etcd, uncomment the following line
+    # and add the certificate in cilium-etcd-secrets bellow
```

Comment:
Can you add a comment when to use this over the secrets?

Reply:
It's not a matter of which one is better. If you uncomment this line bellow, then you need to add the secret on line 41. I'll clarify this a bit.

Reply:
Fixed